src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    Circle(String color, double radius) {
```

Comment:
```suggestion
    public Circle(String color, double radius) {
```
constructor must be public. Fix everywhere